docs/guides/email/email-services/postfix-smtp-debian7/index.md

```diff
@@ -167,7 +167,7 @@ You can now test your Postfix Configurations by using your system's `mail` utili
 
 1. Compose an email to verify that your system is able to successfully send it. Replace the email address with your own and your intended recipient's email address.
 
-        echo  "body of your email" | mail -s "This is a subject" -a "From:you@example.com" recipient@elsewhere.com
+        echo  "body of your email" | mail -s "This is a subject" -A "From:you@example.com" recipient@elsewhere.com
```

Comment:
Did we verify this was a correct recommendation? I did some research, and it looks like the [official documentation](https://mailutils.org/manual/html_section/mail.html#:~:text=content%2Dfilename.-,%2Da%20header%3Avalue,-%2D%2Dappend%3Dheader) for the `mail` command does, in fact, use the lowercase "a" for specifying mail header information. The capital "A" is used for attaching files.

In the example from our guide, it looks like we're specifying header info, not attaching a file.

Reply:
Hi @jddocs  This is what I see in the official docs: https://mailutils.org/manual/html_chapter/Programs.html#Composing-Mail. Looks like even the suggestion that the reader made was incorrect. Please feel free to update the command with the option that you seem fits.